knowledge_repo/app/index.py

```diff
@@ -124,7 +124,34 @@ def index_up_to_date():
             return False
     return True
 
+@ErrorLog.logged
+def update_index_for_post(kp,path):
+    import time
```

Comment:
@abhirathb Remove time library, I guess this is being used to benchmark?

Reply:
I thought I had removed it already in a change. I'll double check